src/tools/dev/scripts/bv_support/bv_vtk.sh

```diff
@@ -2719,11 +2719,11 @@ function build_vtk
 
     # Some linker flags.
     lf=""
-    if test "${OPSYS}" = "Darwin" ; then
-        lf="-Wl,-headerpad_max_install_names"
-        lf="${lf},-compatibility_version,${VTK_COMPATIBILITY_VERSION}"
-        lf="${lf},-current_version,${VTK_VERSION}"
-    fi
+#    if test "${OPSYS}" = "Darwin" ; then
```

Comment:
This PR is about ready. I am still awaiting results of a build after having removed this which I think was causing problems with installed VTK lib names on macOS.

Reply:
Removing this logic did nothing to correct funky naming of VTK libs on macOS

Reply:
Did you try passing ```-DVTK_CUSTOM_LIBRARY_SUFFIX:STRING=""```
That should remove the starting `9.2` from libs with version strings like `9.2.1` and `9.2.9.2.6`.